notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb

```diff
@@ -0,0 +1,1289 @@
+{
```

Comment:
<p>Fix the links to point to your new notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Fixed!</p>